src_erl/Communication_Layer/http_Nerlserver/src/Client/clientStatem.erl

```diff
@@ -278,31 +329,66 @@ training(cast, In = {loss,WorkerName,LossFunction,_Time_NIF}, State = #client_st
   nerl_tools:http_request(RouterHost,RouterPort,"lossFunction", term_to_binary({WorkerName,LossFunction})),
   {next_state, training, State#client_statem_state{myName = MyName,etsRef = EtsRef}};
 
+
+training(cast, In={worker_kill , Body}, State = #client_statem_state{etsRef = EtsRef}) ->
+  ets:update_counter(EtsRef, msgCounter, 1),
+  ets:update_counter(EtsRef, infoIn, nerl_tools:calculate_size(In)),
+  {_, WorkerName} = binary_to_term(Body),
+  Pid = ets:lookup_element(EtsRef, WorkerName, ?WORKER_PID_IDX),
+  gen_statem:stop(Pid, shutdown , infinity),
```

Comment:
What is the meaning of inifinity here?
Please add a comment and let me know what does it mean?